### ISSUE TITLE
Is there any plan for releasing any version with changes in dev?

### ISSUE THREAD

**Issue by jersiovic** [OrchardCMS/Orchard#7922]
I'm worried of using current bits of dev directly to avoid any future problem with an upgrade to an official release.
Is there any schedule or estimated release date  of a new version with changes in dev?
Are my concerns justified in your opinion? Or since last release still there is not any change that needs to add an upgrade process when the new version with changes in dev is going to be released, so I don't have to be worried.

**Comment by sebastienros** [comment]
@BenedekFarkas will help on this.

**Comment by BenedekFarkas** [comment]
All the changes for fixing Dynamic Compilation (and support Roslyn for C# 6 and above) are now working on `1.10.x` (I did some further improvements today). I just merged all those into `dev` and made the necessary changes there to the modules that aren't on `dev` (`Orchard.Glimpse` and `Orchard.OpenId`).

So I think `1.10.x` is ready for a 1.10.3 version release. There's two things on `dev` that we should take care of (and then it's ready for a 1.11 version release):

- https://github.com/OrchardCMS/Orchard/issues/7832#issuecomment-353670873
- #7924
